source/playgrounds.txt

```diff
@@ -30,6 +30,14 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
+|vsce| splits your Playground window to display the results of your 
+Playground in |json| format in the right-side pane labeled ``Playground 
+Results.json``. If you disabled split-view, |vsce| displays the 
+Playground results in |json| format in a new tab to the right of 
+your Playground. You cannot edit the results in the results pane. 
+Instead, you can save the results to a file and edit the results in the 
```

Comment:
What's a scenario where I would need to edit playground results? While what we're saying here is true, I'm a little concerned it may lead to confusion among users who think editing playground results will somehow have an impact on the contents of their database.

I'm wondering if we should remove this last sentence to prevent that possible confusion.